src/libs/Trie/TrieNode.js

```diff
@@ -0,0 +1,50 @@
+
+class TrieNode {
+    constructor() {
+        this.children = {};
+        this.leaf = false;
```

Comment:
Booleans should start with `is`, `has`, or `should`:

```suggestion
        this.isLeaf = false;
```

Reply:
Done.